File: SportVueClient/src/models/tournament.js
export default class Tournament {
    constructor(idTour, nameTour, status, timeEnd, timeStart, type, description, title) {
        this.idTour = idTour;
        this.nameTour = nameTour;
        this.status = status;
        this.timeEnd = timeEnd;
        this.timeStart = timeStart;
        this.type = type;
        this.description = description;
        this.title = title;
    }
}

File: SportsProject/src/main/java/ssv/com/entity/Tournament.java
package ssv.com.entity;

import java.sql.Date;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Tournament {
	private int idTour;
	private String nameTour;
	private int status;
	private Date timeEnd;
	private Date timeStart;
	private String type;
	private String description;
	private String title;
	private List<Schedule> schedule;
	private List<Team> team;
	private int winner;
}
